test/cmdlineTests/linking_strict_assembly_no_file_name_in_link_reference/output

```diff
@@ -0,0 +1,16 @@
+
+======= linking_strict_assembly_no_file_name_in_link_reference/input.yul (EVM) =======
+
+Pretty printed source:
+object "a" {
+    code { let addr := linkersymbol("L") }
+}
+
+
+Binary representation:
+73123456789012345678901234567890123456789050
+
+Text representation:
+  linkerSymbol("8aa64f937099b65a4febc243a5ae0f2d6416bb9e473c30dd29c1ee498fb7c5a8")
```

Comment:
Why does this look different from the input?

Reply:
Because in asm `linkerSymbol()` requires a hex-encoded hash while in Yul `linkersymbol()` it takes a fully qualified library name that gets translated to the hash at the Yul->EVM stage.